### PR TITLE
contrib/godot-export-templates-linux: new package

### PR DESCRIPTION
**Pull request by z-erica**
these are intended to run in non-chimera systems by virtue of being statically linked, keeping all dependencies vendored (matching the upstream templates) and dlopen'ing the system libraries that aren't vendored

### CODE REVIEW

**Comment by nekopsykose** [comment]
you can't dlopen from static binaries

**Comment by nekopsykose** [comment]
so what you're trying to do is actually not possible  

i think having a statically linked libc++ and then trying to dlopen anything that ends up pulling libstdc++ will also not work on top of that (i'm assuming one of said libs are c++)

**Comment by nekopsykose** [comment]
and then on top of that even if the dlopen worked, dlopening a glibc into the same address space (just to be able to open graphics drivers as an example; where mesa drivers also depend on the c++ std) is also broken too

tl;dr you're not the first and this can't work at all unless you're trying to do something different than what it looks like
